src/rtgsfit.c

```diff
@@ -185,17 +189,17 @@ void rtgsfit(
             N_GRID, coef, 1, 0.0, source, 1);   
 
     // modelled measurements
-    cblas_dgemv(CblasRowMajor, CblasTrans,  N_COEF, N_MEAS, 1.0, g_coef_meas_w, 
-            N_MEAS, coef, 1, 0.0, meas_model, 1);       
+    cblas_dgemv(CblasRowMajor, CblasTrans,  N_COEF, N_MEAS, 1.0, g_coef_meas_w_orig, 
+            N_MEAS, coef, 1, 0.0, meas_model, 1);    
             
     // find error between meas and model
     *error = 0.0;
     for (i_meas=0; i_meas<N_MEAS; i_meas++)
     {
-        *error += (meas_no_coil[i_meas] -  meas_model[i_meas]) \
+        *error = *error + (meas_no_coil[i_meas] -  meas_model[i_meas]) \
```

Comment:
I believe if you would do (*error) += it could do what you needed

Reply:
yeah this wasnt actually the issue but your suggestion is nice and explicit